panel/widgets/slider.py

```diff
@@ -319,6 +321,36 @@ def _process_property_change(self, msg):
         return msg
 
 
+class DatetimeSlider(DateSlider):
+    """
+    The DatetimeSlider widget allows selecting a value within a set of
+    bounds using a slider.  Supports datetime.datetime
+    and np.datetime64 values. The step size is fixed at 1 day.
+
+    Reference: https://panel.holoviz.org/reference/widgets/DatetimeSlider.html
+
+    :Example:
+
+    >>> import datetime as dt
+    >>> DatetimeSlider(
+    ...     value=dt.datetime(2025, 1, 1),
+    ...     start=dt.datetime(2025, 1, 1),
+    ...     end=dt.datetime(2025, 1, 7),
+    ...     name="A datetime value"
+    ... )
+    """
+
+    as_datetime = param.Boolean(default=True, readonly=True, doc="""
+        Whether to store the date as a datetime.""")
+
+    step = param.Number(default=60, doc="""
```

Comment:
```suggestion
    step = param.Number(default=60, bounds=(1, None), doc="""
```

I can see `DatetimeRangeSlider` uses step size in ms, though I agree it makes more sense with a second for a slider.